hpOneView/oneview_client.py

```diff
@@ -241,6 +243,13 @@ def switch_types(self):
         return self.__switch_types
 
     @property
+    def logical_switch_groups(self):
```

Comment:
Could you add unit testing for logical_switch_groups? We have some test cases for each resource, you can check some examples at: https://github.com/HewlettPackard/python-hpOneView/blob/master/tests/unit/test_oneview_client.py